arches/management/commands/validate.py

```diff
@@ -281,10 +281,21 @@ def check_integrity(self, check, queryset, fix_action):
 
             if self.options["verbosity"] > 1:
                 self.stdout.write("\t" + "-" * 36)
+                if check.value == 1016:
```

Comment:
The enum is here to avoid using these magic values

```suggestion
                if check.value == IntegrityCheck.TOO_MANY_WIDGETS.value:
```